CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 * By default, `system.abort-jobs-on-terminate` is false when running `java -jar cromwell.jar server`, and true when running `java -jar cromwell.jar run <wdl> <inputs>`.
 * Enable WDL imports when running in Single Workflow Runner Mode.
 * Support for sub workflows
+* Support declarations as graph nodes
```

Comment:
FWIW I was trying to explain this change to person w/o using phrases like "graph node" the other day and struggled. This might warrant some thought.